index.html

```diff
@@ -8,305 +8,357 @@
+              <a href="#html">html</a>
+            </li>
+            <li class="platform">
+              <a href="#ruby">Ruby on rails</a>
+            </li>
+            <li class="platform">
+              <a href="#CSS">css</a>
+            </li>
+            <li class="platform">
+              <a href="#JS">javascript</a>
+            </li>
+          </ul>
+          <button type="button" class="content-view-button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="works">
+      <div class="content">
+        <div class="content-body">
+          <h2 class="content-title">User Navigation</h2>
+          <ul class="layout">
+            <li class="call">Uber</li>
+            <li><img src="images/point.svg" alt="." /></li>
+            <li class="profession">Lead Developer</li>
+            <li><img src="images/point.svg" alt="." /></li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="job-explanation">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="programming">
+            <li class="platform">
+              <a href="#html">html</a>
+            </li>
+            <li class="platform">
+              <a href="#ruby">Ruby on rails</a>
+            </li>
+            <li class="platform">
+              <a href="#CSS">css</a>
+            </li>
+            <li class="platform">
+              <a href="#JS">javascript</a>
+            </li>
+          </ul>
+          <button type="button" class="content-view-button">See Project</button>
+        </div>
+        <img
+          src="images/Snapshoot Portfolio4.png"
+          alt="Work 4"
+          class="content-picture"
+        />
+      </div>
+    </section>
+    <!-- About me section -->
+    <section class="about-me">
+      <div class="page">
+        <h2 class="head">
+          About
+          <br />
+          Myself
+        </h2>
+        <p class="capability">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="social-contact last">LET'S CONNECT</p>
+        <ul class="social-icons">
+          <li>
+            <a href="#twitter">
+              <img src="images/tweeter.png" alt="Twitter icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#linkedin">
+              <img src="images/linkedin.png" alt="Linkedin icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#medium">
+              <img src="images/medium.png" alt="Medium icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#github">
+              <img src="images/github.png" alt="Github icon" />
+            </a>
```

Comment:
- [ ] You did an amazing job with the layout of your 'About me' section :+1:, but to match as much as possible to the Figma template you should make sure that there are 2 blocks, the left one should take 40% and the right one the remaining 60% of the page's width :+1:

**Your design**

![Screenshot from 2023-03-03 08-01-55](https://user-images.githubusercontent.com/105117832/222728264-be6834c1-33b2-47c2-8ef7-272eed2fe0a7.png)

**Your design 175%**

![Screenshot from 2023-03-03 08-11-22](https://user-images.githubusercontent.com/105117832/222728636-4f560e03-bb85-4473-a332-df5c1d6b934c.png)

**Figma**

![Screenshot from 2023-03-03 08-02-16](https://user-images.githubusercontent.com/105117832/222728358-cc7dad6f-dd2a-4a22-b5d5-77b43f7c7924.png)